datacite.gemspec

```diff
@@ -33,4 +33,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "faraday_middleware", "~> 1.0"
   spec.add_dependency "json_schema", "~> 0.21.0"
   spec.add_dependency "zeitwerk", "~> 2.4"
+  spec.metadata["rubygems_mfa_required"] = "true"
```

Comment:
Should we discuss this at a dev planning?

Reply:
sure - add it to the agenda.  I dont even know what it means;  just taking autocorrect.

Reply:
It means that you have to use multifactor authentication with rubygems to publish this gem.

Reply:
I've asked this question twice in #dlss-infrastructure slack channel and no objections so far.  I think we should all be 2-auth for rubygems at this point.